winternlc/create/test.py

```diff
@@ -0,0 +1,212 @@
+import os
+import sys
+from collections import defaultdict
+
+import matplotlib.pyplot as plt
+import numpy as np
+from joblib import Parallel, delayed
+from utils import extract_pixel_values, find_median_files, get_exposure_time
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "../")))
+from config import output_directory, test_directory
+
+LINEAR_REGION_MINCOUNTS = 20000
```

Comment:
If all these scripts get used, then it is bad practise to just have a copy in every script of these variables. What if we want to change them? If the variables are the same thing, you should define them in only one place and then import them into other places. Otherwise there is a big risk of mistakes in future.